.estlintrc.ts

```diff
@@ -9,7 +9,9 @@ export default {
         "@launchware/eslint-config-react",
         "eslint:recommended",
         "plugin:@typescript-eslint/recommended",
-        "@tanstack/query"
+        "@tanstack/query",
+        "airbnb",
```

Comment:
these are probably redundant to what's in https://github.com/LaunchWare/eslint-config-react/blob/main/.eslintrc.cjs so they can be removed.